openmc/mgxs/__init__.py

```diff
@@ -20,6 +20,7 @@
 .. _XMAS-172: https://www-nds.iaea.org/wimsd/energy.htm
 .. _SHEM-361: https://www.polymtl.ca/merlin/downloads/FP214.pdf
 .. _activation: https://fispact.ukaea.uk/wiki/Keyword:GETXS
+.. _VITAMIN-J-42: https://www.oecd-nea.org/dbdata/nds_jefreports/jefreport-10.pdf
```

Comment:
This should be referenced in a description above (perhaps under the activation group structures?)

Reply:
Added to VITAMIN-J-42 activation docstring in order of number of groups